website/docs/concepts/function-components/hooks/use-effect.mdx

```diff
@@ -4,7 +4,7 @@ title: "use_effect"
 
 `use_effect` is used for hooking into the component's lifecycle and creating side-effects.
 
-It takes a function which is called after every component's render finishes.
+It takes a function which is called everty time after the component's render finished.
```

Comment:
```suggestion
It takes a function which is called everty time after the component's render is finished.
```

Reply:
Not so sure about that change. I would either use `has finished` or nothing. But it's up to you, it's really not that important

Reply:
It's fine either way. Use "has" or "is", whichever you want. "render finished" sounds weird